lib/std/json/write_stream.zig

```diff
@@ -140,11 +140,7 @@ pub fn WriteStream(comptime OutStream: type, comptime max_depth: usize) type {
 
         pub fn emitBool(self: *Self, value: bool) !void {
             assert(self.state[self.state_index] == State.Value);
-            if (value) {
-                try self.stream.write("true");
-            } else {
-                try self.stream.write("false");
-            }
+            try std.json.stringify(value, std.json.StringifyOptions{}, self.stream, OutStream.Error, OutStream.write);
```

Comment:
I think this is backwards, std.json.stringify should be utilizing std.json.WriteStream rather than the other way around.

Reply:
It's easy to go from stream to `output(context, ...)` but hard to go in reverse.